src/components/Map.js

```diff
@@ -151,7 +152,8 @@ class Map extends Component {
     });
   }
 
-  flyToGpsLocation = () =>{
+  flyToGpsLocation = () => {
+    ReactGA.event('Map','User pressed location fab');
```

Comment:
location and zoom level ?

Reply:
Max Stephan said zoom level is not needed - location is already tracked by GA